### PR TITLE
[WIP]: Starting work on governance

### PR DESCRIPTION
**Pull request by arfon**
When finished, this PR will capture the Open Journal governance model.

### CODE REVIEW

**Comment by arfon** [comment]
Questions I think we should be addressing here include (there's very probably more):

- What is our policy with respect to the journals. For example, do we mandate open access (I think yes), and open source (I think yes too), and open review (I care less about this one)
- What is the relationship between the editorial board of a journal and the steering committee of Open Journals?
- Who should we invite to be on this steering committee?

**Comment by arfon** [comment]
/ cc @openjournals/joss-editors - please take a look at this and let me know what you think.

**Comment by labarba** [comment]
Open access = absolutely, yes.
Open source = yes
Open review = default to yes, and let any journal proposal that wishes to go closed review, to defend why.

**Comment by labarba** [comment]
Relationship between Open Journals and specific journal editorial boards —> Should be through the EiC.

**Comment by kyleniemeyer** [comment]
I'm with @labarba. I'm leaning towards requiring affiliated journals use open reviews, because I think that's one of the important qualities (a feature, even) of JOSS.

**Comment by labarba** [comment]
Also, operationally, the JOSS infrastructure is designed to work with reviews happening on GitHub issues.  If another journal wanted to close reviews, it sounds like that would require them to use a different platform.

**Comment by alexgarciac** [comment]
Open is great but should not be understood solely as "free". the git issue review model works for software but it does not work for the narrative. I think that part of open journals should be an understanding of the review process as an annotation process and therefore, closed or open, it should be supported by something like web.hypothes.is. In addition, open should also be backed up by linked  data. there is a powerful reason for this and it has to do with query-ability of the content and process-ability  of the content by machines -including bibliographic references, links, etc. There is an implicit desire to have everything wild in the open, I would urge everyone here to also be decentralized by using IPFS and blockchain in support of an open publication workflow -even if just conceptually. Also, an open journal governance should consider the whole publication workflow, not just the actual release. Implicit in my comment it is the understanding that such governance body should acknowledge the release dont publish nature of openness in scientific publications.

**Comment by alexgarciac** [comment]
steering committee should have a wide understanding of openness, be ambitious in addressing issues throughout the publication workflow (pre and post publication, dont publish  but release model), preserve from the git-issue based review model what ever works and make sure that we move into a more annotation-based review model in combination with a git-issue model. steering committee should also communicate to other publishers what is good about using git and why it should be widely used, address the issue of when is this git model not suitable. more importantly, steer and promote more decentralization in the publication workflow. Be provocative and proactive in promoting moving forward instead of becoming a steering committee that preserves the status quo.

File: governance.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Open Journals Governance
+
+## The Project
+
+Open Journals (The Project) is a collection of open source software projects affiliated with the 501c3 NumFOCUS Foundation. The goal of The Project is to develop open source software and deploy open access academic publishing services.
```
Comment (lheagy) [inline governance.md]: minor, but do you want to include the NumFOCUS url? 
```
[NumFOCUS Foundation](https://www.numfocus.org/)
```

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Open Journals Governance
+
+## The Project
+
+Open Journals (The Project) is a collection of open source software projects affiliated with the 501c3 [NumFOCUS Foundation](https://www.numfocus.org). The goal of The Project is to develop open source software and deploy open access academic publishing services.
```
Comment (danielskatz) [inline governance.md]: comment: I think OJ of as more than "a collection of open source software projects"  Isn't it also an operational framework?  (or maybe something else that someone can express better than I am)

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+The software developed by The Project is released under the MIT (or similar) open source license, developed openly and hosted in public GitHub repositories under the [Open Journals GitHub organization](https://github.com/openjournals). Examples of Project Software include the [JOSS web application](https://github.com/openjournals/joss), the [Whedon RubyGem](https://github.com/openjournals/whedon), the [Whedon bot](https://github.com/openjournals/whedon-api) and more.
```
Comment (danielskatz) [inline governance.md]: The software developed and maintained by The Project is released under the MIT (or similar) open source license, developed openly, and hosted in public GitHub repositories under the [Open Journals GitHub organization](https://github.com/openjournals). Examples of Project Software include the [JOSS web application](https://github.com/openjournals/joss), the [Whedon RubyGem](https://github.com/openjournals/whedon), the [Whedon bot](https://github.com/openjournals/whedon-api), and more.

File: governance.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+The services run by The Project consist of public websites and web-services that are hosted under the `theoj.org` and `briefideas.org` domain and include the Journal of Open Source Software (http://joss.theoj.org), the Journal of Open Source Education (http://jose.theoj.org), the Journal of Brief Ideas (http://beta.briefideas.org), and the Open Journal of Astrophysics (http://astro.theoj.org).
```
Comment (danielskatz) [inline governance.md]: The services run by The Project consist of public websites and web services that are hosted under the `theoj.org` and `briefideas.org` domains, and include the Journal of Open Source Software (http://joss.theoj.org), the Journal of Open Source Education (http://jose.theoj.org), the Journal of Brief Ideas (http://beta.briefideas.org), and the Open Journal of Astrophysics (http://astro.theoj.org).

File: governance.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+The Project is formally affiliated with the 501c3 NumFOCUS Foundation (http://numfocus.org), which serves as its fiscal sponsor, may hold project trademarks and other intellectual property, helps manage project donations and acts as a parent legal entity. In addition, one of our journals, the Journal of Open Source Software, is formally affiliated with the Open Source Initiative (https://opensource.org/affiliates/list). NumFOCUS is the only legal entity that has a formal relationship with The Project.
```
Comment (danielskatz) [inline governance.md]: The Project is formally affiliated with the 501(c)(3) NumFOCUS Foundation (http://numfocus.org), which serves as its fiscal sponsor, may hold project trademarks and other intellectual property, helps manage project donations, and acts as a parent legal entity. In addition, currently one of our journals, the Journal of Open Source Software, is formally affiliated with the Open Source Initiative (https://opensource.org/affiliates/list). NumFOCUS is the only legal entity that has a formal relationship with The Project.

File: governance.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+The Project leadership consists of a BDFL (Arfon Smith) and Steering Council.
+
+### BDFL
+
+The Project will have a BDFL (Benevolent Dictator for Life), who is currently Arfon Smith. As Dictator, the BDFL has the authority to make all final decisions for The Project. As Benevolent, the BDFL, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL is chair of the Steering Council (see below) and may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.
```
Comment (danielskatz) [inline governance.md]: The Project will have a BDFL (Benevolent Dictator for Life), who is currently Arfon Smith. As Dictator, the BDFL has the authority to make all final decisions for The Project. The BDFL, being benevolent, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL is chair of the Steering Council (see below) and may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.

File: governance.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+The BDFL can appointing his/her successor, but it is expected that the Steering Council would be consulted on this decision. If the BDFL is unable to appoint a successor, the Steering Council will make a suggestion or suggestions to the Main NumFOCUS Board. While the Steering Council and Main NumFOCUS Board will work together closely on the BDFL selection process, the Main NumFOCUS Board will make the final decision.
```
Comment (danielskatz) [inline governance.md]: The BDFL can appoint his/her successor, but it is expected that the Steering Council would be consulted on this decision. If the BDFL is unable to appoint a successor, the Steering Council will make a suggestion or suggestions to the main NumFOCUS Board. While the Steering Council and main NumFOCUS Board will work together closely in selecting a BDFL, the main NumFOCUS Board will make the final decision.

(comment: the main NumFOCUS Board hasn't been defined.  Is this the board of NumFOCUS itself, or the NumFOCUS Subcommittee below?)

File: governance.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+### Steering Council
+
+The Project will have a Steering Council that is elected to further the overall mission of the The Project. Initially the Council was created by the BDFL, however, future membership rules will determined by the Council itself. The overall role of the Council is to ensure, through working with the BDFL and taking input from the Community, the long-term well-being of the project, both technically, editorially, and as a community.
```
Comment (danielskatz) [inline governance.md]: The Project will have a Steering Council that is elected to further the overall mission of the The Project. The initial Steering Council was created by the BDFL, however, future membership rules will determined by the Steering Council itself. The overall role of the Steering Council is to ensure, through working with the BDFL and taking input from the community, the long-term well-being of the project, both technically, editorially, and as a community.

(comment: this is the first time community was mentioned.  It probably should be defined.  Is it the submitters and readers of all articles in all OJ journals? Or more than that?)

File: governance.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+The Steering Council and its Members play a special role in certain situations. In particular, the Council may:
+
+- Make decisions about the overall scope, vision and direction of The Project.
```
Comment (danielskatz) [inline governance.md]: - Make decisions about the overall scope, vision, and direction of The Project.

File: governance.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+- Make decisions about the Services that are run by The Project and manage those Services for the benefit of the Project and Community.
+- Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+
+#### Steering Council membership
+
+To be written by the Council.
+
+#### Subcommittees
+
+Question: Does each journal need its own subcommittee or is this effectively the editorial board of that journal?
```
Comment (danielskatz) [inline governance.md]: comment: didn't we talk about a board made of the EICs of each journal?

File: governance.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+#### NumFOCUS Subcommittee
+
+The Council will maintain one narrowly focused subcommittee to manage its interactions with NumFOCUS.
+
+- The NumFOCUS Subcommittee is comprised of 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the full Council.
+- This Subcommittee shall NOT make decisions about the direction, scope or technical direction of the Project.
```
Comment (danielskatz) [inline governance.md]: - This Subcommittee shall NOT make decisions about the direction, scope, or technical direction of the Project.

File: governance.md (hunk 2 above)
Comment (arfon) [reply governance.md]: I think you're right. I'm not sure what the correct phrase is though...

File: governance.md (hunk 7 above)
Comment (arfon) [reply governance.md]: > comment: the main NumFOCUS Board hasn't been defined. Is this the board of NumFOCUS itself, or the NumFOCUS Subcommittee below?

I lifted this directly from the Jupyter governance and they have the same inconsistency. I'm inclined to say that this is the NumFOCUS Subcommittee (as currently defined)?

File: governance.md (hunk 10 above)
Comment (arfon) [reply governance.md]: > comment: didn't we talk about a board made of the EICs of each journal?

Yes, I think we decided the EICs of each journal would be in a separate group.

File: governance.md (hunk 2 above)
Comment (tracykteal) [reply governance.md]: Maybe "The Open Journals project develops and deploys open source software for open access publishing and enables publication credit for software and other research and educational products. It is a fiscally sponsored project of the 501(c)3 NumFOCUS."

I'm not sure if you need to put the fiscal sponsorship in the governance docs though.

File: governance.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+The Project will have a Steering Council that is elected to further the overall mission of the The Project. The initial Steering Council was created by the BDFL, however, future membership rules will determined by the Steering Council itself. The overall role of the Steering Council is to ensure, through working with the BDFL and taking input from the community, the long-term well-being of the project, both technically, editorially, and as a community.
+
+The Steering Council and its Members play a special role in certain situations. In particular, the Council may:
+
+- Make decisions about the overall scope, vision, and direction of The Project.
+- Make decisions about strategic collaborations with other organizations or individuals.
+- Make decisions about new starting a new journal (or incorporating an existing journal) under the Open Journals organization.
+- Make decisions about the Services that are run by The Project and manage those Services for the benefit of the Project and Community.
+- Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+
```
Comment (tracykteal) [inline governance.md]: Likely should add in something about financial oversight and planning. This would include being able to apply for grants too. 
- Provide financial oversight and planning for the The Project

File: governance.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+Question: Does each journal need its own subcommittee or is this effectively the editorial board of that journal?
+
+#### NumFOCUS Subcommittee
+
+The Council will maintain one narrowly focused subcommittee to manage its interactions with NumFOCUS.
+
+- The NumFOCUS Subcommittee is comprised of 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the full Council.
+- This Subcommittee shall NOT make decisions about the direction, scope, or technical direction of the Project.
+- This Subcommittee will have 5 members, 4 of whom will be current Steering Council members and 1 of whom will be external to the Steering Council.
```
Comment (tracykteal) [inline governance.md]: So, this is a separate subcommittee from the Steering Council? And it primarily does financial oversight and planning? I'm not quite sure why it's set up this way, rather than a responsibility of the Steering Council or as a subcommittee of the Steering Council, or a Treasurer. Is it that this committee will do the day-to-day work, and you want to separate out that responsibility? Totally understand there could be a reason here, just thinking this could make things more difficult to have operationally aligned.

Comment (tracykteal) [reply governance.md]: Also, rather than 'NumFOCUS Subcommittee' a more generalizable name would be good for governance names, like 'Fiscal Subcommittee' or 'Operations Subcommittee'

File: governance.md (hunk 12 above)
Comment (tracykteal) [reply governance.md]: I see below this has been separated out into another subcommittee, so just wondering where it fits best.

File: governance.md (hunk 13 above)
Comment (arfon) [reply governance.md]: > Totally understand there could be a reason here, just thinking this could make things more difficult to have operationally aligned.

There actually isn't. This is language I lifted directly from the Jupyter governance documents.

Perhaps the fiscal subcommittee/NumFOCUS subcommittee should simply be a subset of the main steering council?

Comment (tracykteal) [reply governance.md]: I think having it be a subset of the main steering council makes the most sense. I suggested a line about fiscal oversight in the steering committee role, so it could be included there. Then the Steering Council could decide if they want a subcommittee or a Treasurer to have this role.

Comment (karthik) [reply governance.md]: +1 to what Tracy said. Fiscal oversight could be handled by a subset of steering and may not necessarily warrant a separate subcommittee. So having the option to do one or other would be better.

**Comment by Kevin-Mattheus-Moerman** [comment]
Minor comments. 

1. What is meant by "similar" in this sentence:
>  the MIT (or similar) open source license   

Do we mean a similar type of license, i.e. a permissive type license like BSD or Apache? Or do we mean any OSI approved license in general? It is probably the latter so perhaps this should  be rephrased to something like:

>  an [Open Source Initiative approved open source license](https://opensource.org/licenses) (e.g. the [MIT](https://opensource.org/licenses/MIT) license)

2. I would suggest rewording:

> ... the Journal of Open Source Software, is formally affiliated with the Open Source Initiative

To: 

> ... the Journal of Open Source Software, is an affiliate member of the Open Source Initiative

as I think it is more correct to say what we are exactly, an affiliate member, since formally affiliated could mean lots of things. 

3. The word "new" appears twice in:

> Make decisions about new starting a new journal...

**Comment by tracykteal** [comment]
On the call it came up, that there should be a component about the individual journals under Open Journals. Something like:

"Individual journals are responsible for their own editorial process, including their editorial board composition"

**Comment by arfon** [comment]
OK thanks for the feedback all. I've tried to incorporate all of your feedback.

I'd like an explicit 👍from the proposed steering council members before merging this:

- Lorena Barba (@labarba)
- Katy Huff (@katyhuff)
- Dan Katz (@danielskatz)
- Karthik Ram (@karthik)
- Kevin Moerman (@Kevin-Mattheus-Moerman)
- Kyle Niemeyer (@kyleniemeyer)

File: governance.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
+# Open Journals Governance
+
+## The Project
+
+Open Journals (The Project) develops and deploys open source software for open access publishing and enables publication credit for software and other research and educational products. It is a fiscally sponsored project of the 501(c)(3) [NumFOCUS Foundation](https://www.numfocus.org).
+
+The software developed and maintained by The Project is released under the MIT (or similarly permissive) open source license, developed openly, and hosted in public GitHub repositories under the [Open Journals GitHub organization](https://github.com/openjournals). Examples of Project Software include the [JOSS web application](https://github.com/openjournals/joss), the [Whedon RubyGem](https://github.com/openjournals/whedon), the [Whedon bot](https://github.com/openjournals/whedon-api), and more.
```
Comment (danielskatz) [inline governance.md]: With "include ...", we don't need "and more"

File: governance.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
+# Open Journals Governance
+
+## The Project
+
+Open Journals (The Project) develops and deploys open source software for open access publishing and enables publication credit for software and other research and educational products. It is a fiscally sponsored project of the 501(c)(3) [NumFOCUS Foundation](https://www.numfocus.org).
```
Comment (danielskatz) [inline governance.md]: Can we add something about coordinating the operations of the Open Journals, or something similar?

File: governance.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+
+The Project leadership consists of a BDFL (Arfon Smith) and Steering Council.
+
+### BDFL
+
+The Project will have a BDFL (Benevolent Dictator for Life), who is currently Arfon Smith. As Dictator, the BDFL has the authority to make all final decisions for The Project. The BDFL, being benevolent, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL is chair of the Steering Council (see below) and may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.
```
Comment (danielskatz) [inline governance.md]: everything up to this point has been present tense, so I think much of this should be too:
"The Project has a BDFL..."

File: governance.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+
+The BDFL can appoint his/her successor, but it is expected that the Steering Council would be consulted on this decision. If the BDFL is unable to appoint a successor, the Steering Council will make a suggestion or suggestions to the main NumFOCUS Board. While the Steering Council and main NumFOCUS Board will work together closely in selecting a BDFL, the main NumFOCUS Board will make the final decision.
```
Comment (danielskatz) [inline governance.md]: wording again: "The BDFL appoints his/her successor, and it is expected that the Steering Council is consulted on this decision. If the BDFL does not appoint a successor, the Steering Council will make a suggestion or suggestions to the main NumFOCUS Board. While the Steering Council and main NumFOCUS Board will work together closely in selecting a BDFL, the main NumFOCUS Board will make the final decision."

Also, the "main NumFOCUS Board" hasn't been define here.

File: governance.md [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+
+### Steering Council
+
+The Project will have a Steering Council that is elected to further the overall mission of the The Project. The initial Steering Council was created by the BDFL, however, future membership rules will determined by the Steering Council itself. The overall role of the Steering Council is to ensure, through working with the BDFL and taking input from the community, the long-term well-being of the project, both technically, editorially, and as a community.
```
Comment (danielskatz) [inline governance.md]: tenses again: "The Project has a Steering Council ..."

File: people.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
...
+## Benevolent Dictator for Life
+
+Arfon Smith (@arfon) is the Benevolent Dictator for Life (BDFL).
+
+## Steering Council
+
+- Chair: Arfon Smith (@arfon)
+- Member: Lorena Barba (@labarba)
+- Member: Katy Huff (@katyhuff)
+- Member: Dan Katz (@danielskatz)
```
Comment (danielskatz) [inline people.md]: Daniel S. Katz please

File: governance.md (hunk 17 above)
Comment (tracykteal) [reply governance.md]: Open Journals is its own project, so NumFOCUS wouldn't be involved in selection of project leadership. 

"The BDFL can appoint his/her successor, but it is expected that the Steering Council would be consulted on this decision. If the BDFL is unable to appoint a successor, the Steering Council will appoint a successor with a majority vote."

I'm not sure if 'majority vote' is what you would want, but it should be something internal to the project.

**Comment by arfon** [comment]
Thanks for the extra comments/changes @danielskatz and @tracykteal. I've gone ahead and incorporated your edits.

**Review by Kevin-Mattheus-Moerman — COMMENTED** [review]
Looks good. Minor typo "... future membership rules will [be] determined by..."

**Comment by danielskatz** [comment]
Thanks @arfon - I've added a 👍 above.

**Comment by Kevin-Mattheus-Moerman** [comment]
@arfon is there a reason we limit ourselves to permissive licenses for "The Project"?  

> The Project is released under the MIT (or similarly permissive) open source license

Also, by saying that the Project carries an MIT license, are you kind of saying that future versions will too? Of course the only way to enforce that future versions will remain open source is to use a copy left rather than a permissive license. 

[See this wiki page on permissive licenses](https://en.wikipedia.org/wiki/Permissive_software_licence): 

> Permissive licenses do not try to guarantee that future generations of the software will remain free and publicly available, in contrast to licenses which have reciprocity requirements which try to enforce this. 

Some groups and journals to be developed might prefer a copy left license to "ensure" it stays open source, e.g. a GPL license ([this being the strongest](https://opensource.org/licenses/AGPL-3.0)). I do not feel too strongly about this but the above is why I tend to favor GPL licenses. 

If you/we have good arguments for it I do not mind if it stays MIT/permissive but if you/we have no preference perhaps we can discuss the license. If we have no preference at this points perhaps we could also not be restrictive and just say "an Open Source Initiative approved open source license"?

**Comment by arfon** [comment]
> If you/we have good arguments for it I do not mind if it stays MIT/permissive but if you/we have no preference perhaps we can discuss the license. If we have no preference at this points perhaps we could also not be restrictive and just say "an Open Source Initiative approved open source license"?

I actually do have a preference for MIT/BSD/permissive. I'm pretty much aligned with the ideas expressed by John Hunter (matplotlib-fame) here: http://nipy.sourceforge.net/nipy/stable/faq/johns_bsd_pitch.html

**Comment by arfon** [comment]
@kyleniemeyer @labarba @Kevin-Mattheus-Moerman can I get a :+1: from you all on this PR?

**Comment by labarba** [comment]
👍 from me, but it looks like a couple of typos noted by @Kevin-Mattheus-Moerman remain; for example "future membership rules will determined by the Steering Council itself" (missing a "be"), and the extra "new" elsewhere.

**Comment by arfon** [comment]
🎉 thanks everyone!
